sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -265,10 +266,22 @@ final class DataFrameWriter private[sql](df: DataFrame) {
    * @param connectionProperties JDBC database connection arguments, a list of arbitrary string
    *                             tag/value. Normally at least a "user" and "password" property
    *                             should be included.
+   * @param columnMapping Maps DataFrame column names to target table column names.
+   *                           This parameter can be omitted if the target table has/will be
```

Comment:
I had followed the example given in line 103:
  def options(options: scala.collection.Map[String, String]): DataFrameWriter = {
and assumed that this is to emphasize the use of a *Scala* Map instead of a *Java* Map.

Should I replace any use of 

scala.collection.immutable.Map[String, String]

by 

Map[String, String]

?